topics/admin/tutorials/pulsar/tutorial.md

```diff
@@ -72,19 +73,152 @@ In this tutorial, we will:
 This tutorial assumes that you have:
 
 - A VM or machine where you will install Pulsar, and a directory in which the installation will be done. This tutorial assumes it is `/mnt`
-- That you have completed the "Galaxy Installation with Ansible" and CVMFS tutorials (Job configuration tutorial is optional.)
+- That you have completed the "Galaxy Installation with Ansible" and CVMFS tutorials (Job configuration tutorial is optional) and have access to the VM/computer where it is installed.
+
+# Overview
+
+We will be installing the RabbitMQ server daemon onto the Galaxy server to act as an intermediary message passing system between Galaxy and the remote Pulsar. The figure below shows a schematic representation of the system.
+
+![pulsar_australia.png](../../images/pulsar_amqp_schema.png)
```

Comment:
do you have an SVG for this figure if we want to update it? Also could you write something more explicit for the figure caption? [Guidlines on alttext](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text)